OWNERS

```diff
@@ -3,5 +3,7 @@
 approvers:
  - dhellmann
  - hardys
+ - maelk
```

Comment:
Mael has done great work in the cluster-api provider code, but I don't know about this repo. Let's add 1 person at a time.

Reply:
I was thinking it may be useful if @maelk has commit rights here to fast-fix any CI regressions with may occur impacting nordixinfra CI, as has happened recently.

Reply:
That's a good point. @stbenjam if you go ahead and add @maelk back to the patch, we can add both of you together.

Reply:
Done